.gitignore

```diff
@@ -16,11 +16,20 @@ Makefile.in
 *.la
 .libs
 /build-aux
+/CMakeCache.txt
+/CMakeDoxyfile.in
+/CMakeDoxygenDefaults.cmake
+/CMakeFiles/
+/cmake_install.cmake
+/CTestTestfile.cmake
```

Comment:
Alright!

Reply:
I think that don't add /build/ in the file . It can add to git globel config. beacuse other man don't use build, he can use build_windows for windows, use build_linux for linux